k8s/object.bzl

```diff
@@ -173,7 +173,7 @@ _common_attrs = {
     # This is only needed for describe.
     "kind": attr.string(),
     "image_chroot": attr.string(),
-    "_resolver": attr.label(
+    "resolver": attr.label(
```

Comment:
Can you add this to the `k8s_object` signature documentation at the bottom of the `README.md`?

Reply:
Done.